scripts/run_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -x
+
+export PYTHONUNBUFFERED=1
+BASE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ "$CI" == "true" ]; then
+    pip3 install coverage pytest-xdist
+
+    set -e
+    # Wait until database is ready
+    $BASE_DIR/wait-for-it.sh ${DATABASE_HOST:-db}:${DATABASE_PORT-5432}
```

Comment:
let's use the wait-for-it package. apt-get install wait-for-it and use `wait-for-it ${....`